app/jobs/mail_receive_job.rb

```diff
@@ -3,34 +3,39 @@
 class MailReceiveJob < ApplicationJob
   queue_as :mail_handlers
 
-  def perform(recipients, message_url) # rubocop:disable Metrics/MethodLength
+  def perform(recipients, message_url)
     fetched_mail = MailgunFetcher::Mail.new(message_url)
     mail_aliases = gather_valid_aliases(recipients, fetched_mail)
     return unless mail_aliases.any?
 
     mail_aliases.each do |mail_alias|
-      if mail_alias.moderation_type == 'open' || (mail_alias.moderation_type == 'semi_moderated' &&
-        mail_alias.mail_addresses.include?(fetched_mail.sender))
-        MailForwardJob.perform_later(mail_alias, message_url)
-      else
-        send_mail_moderations(mail_alias, message_url, fetched_mail)
-      end
+      MailForwardJob.perform_later(mail_alias, message_url)
       mail_alias.update(last_received_at: Time.zone.now)
-    end
-  end
 
-  def send_mail_moderations(mail_alias, message_url, fetched_mail)
-    stored_mail = StoredMail.create(message_url: message_url, received_at: fetched_mail.received_at,
-                                    sender: fetched_mail.sender, mail_alias: mail_alias,
-                                    subject: fetched_mail.subject)
-
-    mail_alias.moderators.each do |moderator|
-      MailModerationMailer.request_for_moderation_email(moderator, stored_mail).deliver_later
+      # if mail_alias.moderation_type == 'open'
```

Comment:
I think this can be removed, right? That's why we use Git :)

Reply:
Good point :)